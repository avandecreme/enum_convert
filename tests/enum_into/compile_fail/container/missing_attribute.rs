use enum_convert::EnumInto;

#[derive(EnumInto)]
// Missing #[enum_into(Target)] attribute
enum Source {
    Unit,
}

enum Target {
    Unit,
}

fn main() {}
