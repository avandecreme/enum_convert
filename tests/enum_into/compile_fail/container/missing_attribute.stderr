error: proc-macro derive panicked
 --> tests/enum_into/compile_fail/container/missing_attribute.rs:3:10
  |
3 | #[derive(EnumInto)]
  |          ^^^^^^^^
  |
  = help: message: enum_into attribute with target enum names is required
